README.md

```diff
@@ -72,6 +75,23 @@ npm publish --registry http://localhost:4873
 
 > **NOTE**: you need a fresh login, so that verdaccio recognizes your owned groups
 
+## Authentication Cache
+
+In order to avoid too many authentication requests to the underlying gitlab instance, the plugin provides an in-memory cache that will save the detected groups of the users for a configurable ttl in seconds. No clear-text password will be saved in-memory, just an SHA-256 hash and the groups information.
+
+By default, the cache will be enabled and the credentials will be stored for 300 seconds. The ttl is checked on access, but there's also an internal timer that will check expired values regularly, so data of users not actively interacting with the system will also be eventually invalidated.
```

Comment:
The ttls will be always enforced on access. In order to cleanup in-memory entries for inactive users, it uses the defaults of node-cache (check each 600 seconds). It could be made configurable, but might be overkill.